challenges/objects-arrays.js

```diff
@@ -6,120 +6,225 @@
   object name, diet, weight, length, period
 */
 
+class Dinosaur {
```

Comment:
Great job creating the dinosaur class rather than just creating the objects 1 by 1!